esrally/track/params.py

```diff
@@ -580,6 +580,21 @@ def percent_completed(self):
         return self.current_bulk / self.total_bulks
 
 
+class ForceMergeParamSouce:
```

Comment:
Furthermore, can you please derive this class from the class `ParamSource`? All Rally-internal parameter sources are derived from it and it provides a default implementation for required methods. Currently, the following invocation:

```
esrally --test-mode --distribution-version=7.4.2 --challenge=append-no-conflicts-index-only --on-error=abort
```

is failing with:

```
[ERROR] Cannot race. Error in load generator ('ForceMergeParamSouce' object has no attribute 'partition')
```

This can be solved by deriving your class from `ParamSource` as I've outlined above.

Reply:
Done.